distribution/docker/solr-cloud/README.md

```diff
@@ -0,0 +1,35 @@
+# DDF Compatible Solr Cloud Containers
+
+This docker compose build two images: a zookeeper and an official solr image
+The solr image has a few add-on components to make it fully compatible with DDF
+
+## Usage
+
+To run: `docker-compose up`
+
+## Running DDF in Cloud mode
+
+Prior to start up DDF, edit the custom.system.properties file
+
+```
+start.solr=false
```

Comment:
❓ Would it not be worth scripting instead of having folks remember to edit config file? I can see this leading to a lot of confusion and questions/statements why/that it is not working.

Reply:
i am debating on that. DDF and Solr Cloud are two decoupled system. Even if there is a script. they will still have to remember to run it when switching DDF between local/cloud mode. Not sure if it is worth maintaining another script.